tensorflow/tensorflow.bzl

```diff
@@ -256,7 +256,7 @@ def tf_cuda_cc_tests(tests, deps, tags=[], size="medium"):
 def tf_gpu_kernel_library(srcs, copts=[], cuda_copts=[], deps=[], hdrs=[],
                           **kwargs):
   cuda_copts = ["-x", "cuda", "-DGOOGLE_CUDA=1",
-                "-nvcc_options=relaxed-constexpr", "-nvcc_options=ftz=true",
+                "-nvcc_options=expt-relaxed-constexpr", "-nvcc_options=ftz=true",
```

Comment:
Have you tried seeing whether this breaks earlier nvcc's?  I don't know if at this point we're willing to break everybody but nvcc 7.5, given how many people have difficulty getting admins to upgrade software on clusters ;)